#145026979 Content for landing page

efalayi opened the pull request:
#### What does this PR do?
This PR fixes signup/signin redirect link and adds 'play game' buttons to index banner

#### Description of Task to be completed?
- change authentication redirect link to home page
- display 'play game' buttons in a prominent part of the index page

#### How should this be manually tested?
N/A

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
* #145026975
* #145026973
* #145026981

Commit:
- add 'play with' buttons to banner
- change signin/signup redirect route to home page
[Fixes #145026979]

andela-moe commented on app/controllers/users.js:
```diff
@@ -30,6 +31,7 @@ exports.signin = (req, res) => {
  * Show sign up form
  */
 exports.signup = (req, res) => {
+  console.log('came here');
```
Please respond to this. No console statement

andela-moe commented on app/controllers/users.js:
```diff
@@ -20,6 +20,7 @@ exports.authCallback = (req, res) => {
  */
 exports.signin = (req, res) => {
   if (!req.user) {
+    console.log('came here');
```
Please respond to this. No console statement

andela-moe reviewed (commented):
Please address

Commit:
- fix page reload on signup/signin
- fix selected avatar error
- clear token from local storage on signout
git commit -m

Commit:
fix lint errors

Commit:
fix eslint error
